test/functional/test_framework/mininode.py

```diff
@@ -299,6 +299,7 @@ def on_reject(self, message): pass
     def on_sendcmpct(self, message): pass
     def on_sendheaders(self, message): pass
     def on_tx(self, message): pass
+    def on_notfound(self, message): pass
```

Comment:
Is this change related to this PR?

Reply:
no, this code is not necessary. will remove it.